src/shared/config.ts

```diff
@@ -33,6 +33,6 @@ export const emDash = "\u2014";
  * /u/username@server.com
  */
 export const instanceLinkRegex =
-  /(\/[c|m|u]\/[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}|![a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,})/g;
+  /(\/[c|m|u]\/|!)[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}/g;
```

Comment:
One last alteration. regex `[c|m|u]` will match the OR pipes `|` literally, so /|/community@instance.tld will match as is.
Replace `[c|m|u]` with `[cmu]`. You could also rearrange the letters for an inappropriate easter egg

Reply:
Good catch, updated. Decided against the easter egg lol